tuf/client/updater.py

```diff
@@ -130,6 +130,7 @@
 import warnings
 
 from securesystemslib import exceptions as sslib_exceptions
+from securesystemslib import formats as sslib_formats
```

Comment:
Can we just import the formats we care about?
`from securesystemslib import PATH_SCHEMA, NAME_SCHEMA, ...` ?

Reply:
probably reasonable, will test this for sslib_formats and sslib_exceptions

Reply:
shoot, I had missed this suggestion in this pass. This is not done yet...

Reply:
Decided not to do this after all: in some cases this approach would work but there are a lot of uses in repository_tool, repository_lib, keydb and roledb . Some of them would end up with clashes (at least exceptions.Error)... The schemas probably would work but I'm not sure the amount of work is reasonable if we want to change all uses.

I think I would prefer leaving this outside of the PR.

Reply:
Sounds reasonable to me.